src/features/marketplace/process/clients/marketplace-client-processor.ts

```diff
@@ -0,0 +1,21 @@
+import MarketplaceClientCreator from './marketplace-client-creator';
+import MarketplaceState from '../types/marketplace-state';
+import constants from '../../../../common/configuration/constants';
+import BusinessError from '../../../../common/exceptions/business-error';
+
+class MarketplaceClientProcessor {
+  async execute(state: MarketplaceState): Promise<MarketplaceState> {
+    const client = MarketplaceClientCreator.createClient(state);
+
+    const response = await client.execute();
+
+    if (response.error) {
+      const { message, code } = constants.errors.minting[response.error.toString()];
```

Comment:
You should create another object for listing errors and not mixing them with the minting ones